examples/layers/src/app.js

```diff
@@ -0,0 +1,67 @@
+/* global window */
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+import MapGL from 'react-map-gl';
+import StyleControls from './style-controls';
+
+const token = process.env.MAPBOX_ACCESS_TOKEN; // eslint-disable-line
```

Comment:
Also check for `MapboxAccessToken` here? Since the `webpack` config is looking for both.